google/auth/transport/__init__.py

```diff
@@ -29,9 +29,18 @@
 import six
 from six.moves import http_client
 
+DEFAULT_RETRYABLE_STATUS_CODES = (
```

Comment:
Where is 429?

Reply:
Python2's six library does not contain a 429 code. I can add one manually as a solution, or we can add it once Python 2 support is dropped